chapters/VK_KHR_surface/wsi.txt

```diff
@@ -1010,18 +1043,10 @@ the following color spaces:
 | AdobeRGB       | 0.640, 0.330 | 0.210, 0.710  | 0.150, 0.060 | 0.3127, 0.3290 (D65) | AdobeRGB
 |====
 
-The transfer functions are described in the "Transfer Functions" chapter of
-the <<data-format,Khronos Data Format Specification>>.
-
-// @@@ The ref page is ended earlier than in the original markup due to
-// asciidoctor issues described in internal MR 2201.
-endif::VK_EXT_swapchain_colorspace[]
-
---
-
-ifdef::VK_EXT_swapchain_colorspace[]
```

Comment:
Does this ifdef need to stay to match the endif below?

Reply:
No, it is merged with previous ifdef for which I removed endif at L1018